source/Halibut.TestUtils.CompatBinary.Base/Services/EchoService.cs

```diff
@@ -2,8 +2,9 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using Halibut.TestUtils.Contracts;
 
-namespace Halibut.TestUtils.Contracts
+namespace Halibut.TestUtils.SampleProgram.Base.Services
 {
     public class EchoService : IEchoService
```

Comment:
@nathanwoctopusdeploy Just a thought. But is it clear enough that this `EchoService` is actually an implementation that uses legacy Halibut?

It's in the `Halibut.TestUtils.CompatBinary.Base`, but is that enough. It could be.

The thing I was wondering was would it make things clearer if we renamed the service as well? E.g. `LegacyEchoService` (or something similar)

Reply:
I would leave as it is, if we find people getting confused we can revisit